### ISSUE TITLE
[bug]: do not use this, its bad

### ISSUE THREAD

**Issue by thq919** [Stacked-Org/stacked#964]
### Describe the bug
programming
i never saw library badder than that (When I say that I mean not this library)

### To reproduce

be developer (Go to [stacked](https://stacked.filledstacks.com/), install cli, watch magic happen) 

### Expected behavior

do not use it (unless you want a really awesome and maintainable product)

### Screenshots

go for https://stacked.filledstacks.com/ (and have your mind blown)

### Additional Context

If issue seems weird, the developer is a pretty angry, racist, low IQ developer and made some really nasty remarks. I figured I'd leave the post here with his/her original posts and have some fun with it.

**Comment by ynnob** [comment]
**I got really angry and smashed my keyboard.**
I completely lost CTRL.

If you can't figure out how to use Stacked you are welcome to ask questions. But you should rethink the way you act in this space.

**Comment by FilledStacks** [comment]
What a terrible thing to do. 

@ynnob I wish people like @thq919 would keep to themselves and struggle with their lack of programming and comprehension skills outside of a space where people are building free tools to share knowledge.

**Comment by ferrarafer** [comment]
Someone with a lot of free time 🤣 .

Criticism is always welcome as long as it is well-founded.

**Comment by thq919** [comment]
i dont give a f about english, i dont give a f about you indian thinking about my developer skills, only thing that matter, that you thinking that is best way to write  code to use your library butt plug.  And I came to dispel your illusions little Hindu.
I ran into your library in commercial code that I need to maintain and it's the worst shit I've ever seen.

And i cant do anything about it, coz i got paid for maintain that, i cant change the way its written by core. 
MVC is brain cancer itself, and your library just a butt plug that force me to write like im fucking android developer. 

I dont need any advice and help.
I just need you to know that override flutter widget system dont make your library - best way to code.
Its just bad wrapper under flutter ecosystem,  that dont add anything comfortable, but make your code more limited.

In overall:
I have great experience with CA, and your library that implements MVC and force me to do so - is a nightmare for me.
